src/BrightScriptDebugSession.ts

```diff
@@ -319,6 +330,14 @@ export class BrightScriptDebugSession extends DebugSession {
         }
     }
 
+    /**
+     * Reusable function just to keep things DRY
+     */
+    private successfulDeployPreparations(response: DebugProtocol.LaunchResponse, host: string) {
```

Comment:
I don't think the abstraction is worth the effort in this case. This function is only called in 2 places, and there are only 2 lines in this function, so it's a little more readable to just inline them and remove this function.